layouts/error.vue

```diff
@@ -1,44 +1,279 @@
-<template>
-  <div>
-    <breadcrumb-trail :breadcrumb="breadcrumb" :title="pageTitle" />
-    <div class="page-wrap container">
-      <div class="subpage">
-        <div>
-          <h4>
-            PAGE NOT FOUND
-          </h4>
-          <div>
-            <p>
-              I am sorry but we cannot find what you have been looking for.
-            </p>
-            <p>        
-              Maybe the page has moved - we suggest visiting the homepage to help you discover what you are looking for.
-            </p>
-          </div>
-        </div>
-      </div>
-    </div>
-  </div>
-</template>
-
-<script>
-import graphcmsQuery from '@/services/graphcmsQuery'
-
-export default {
-  name: 'ErrorPage',
-
-  data: () => {
-    return {
-      pageTitle: 'PAGE NOT FOUND',
-      breadcrumb: [
-        {
-          to: {
-            name: 'index'
-          },
-          label: 'HOME'
-        }
-      ],
-    }
-  }
-}
-</script>
\ No newline at end of file
+<template>    
+  <div class="error-page">
+    <breadcrumb-trail :breadcrumb="breadcrumb" :title="pageTitle" />
+    <div class="page-wrap-container">
+        <h4>
+          PAGE NOT FOUND
+        </h4>
+        <div class="sub-container">
+          <div class="error-left-subcontainer">
+            <div class="error-content">
+            <p>
+              I am sorry but we cannot find what you have been looking for.
+            </p>
+            <p>        
+              Maybe the page has moved - we suggest visiting the homepage to help you discover what you are looking for.
+            </p>
+            </div>
+            <div class="btn-backhome">
+              <el-button @click="backToHome()">VISIT THE HOMEPAGE</el-button>

```

Comment:
Use nuxtlink instead - https://nuxtjs.org/docs/features/nuxt-components/#the-nuxt-component

Reply:
Did you take a look at replacing this with nuxt-link?